### PR TITLE
fix: a few certs issues

### PR DESCRIPTION
**Pull request by alejandroEsc**
Should fix some known issues with the certificate, nothing major here.

### CODE REVIEW

File: charts/redpanda/templates/certs.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ limitations under the License.
   {{- $listeners := .Values.listeners -}}
   {{- $values := .Values }}
   {{- range $name, $data := .Values.tls.certs }}
-    {{- $d := $data.duration }}
+  {{- $d := $data.duration }}
```
Comment (joejulian) [inline charts/redpanda/templates/certs.yaml:25]: We indent here because it's part of the range loop

File: charts/redpanda/templates/certs.yaml [hunk 2]
```diff
@@ -47,25 +48,26 @@ spec:
...
   - "*.{{ $values.external.domain }}"
 {{- end }}
   duration: {{ $d | default "43800h" }}
   isCA: false
-  commonName: {{ template "redpanda.fullname" $ }}-{{ $name }}-cert
   secretName: {{ template "redpanda.fullname" $ }}-{{ $name }}-cert
   privateKey:
     algorithm: ECDSA
     size: 256
-    {{- if not (empty $data.issuerRef) }}
```
Comment (joejulian) [inline charts/redpanda/templates/certs.yaml:60]: same note about indentation for each of these.

**Review by joejulian — APPROVED** [review]
Approved, but with indentation nits

**Comment by alejandroEsc** [comment]
I'll fix the nits before meeting.

**Comment by alejandroEsc** [comment]
I'll fix the nits before meeting.

**Pull request merged**